apps/gateway/src/routes/api/v1/chats/handlers/addMessage.ts

```diff
@@ -22,17 +23,26 @@ export const addMessageHandler = factory.createHandlers(
     return streamSSE(
       c,
       async (stream) => {
-        const { documentLogUuid, messages, source } = c.req.valid('json')
+        const { uuid, messages } = c.req.valid('json')
         const workspace = c.get('workspace')
 
         const result = await addMessages({
           workspace,
-          documentLogUuid,
+          documentLogUuid: uuid,
           messages,
-          source,
+          source: LogSources.API,
         }).then((r) => r.unwrap())
 
-        await pipeToStream(stream, result.stream)
+        let id = 0
+        for await (const event of streamToGenerator(result.stream)) {
+          const data = chainEventPresenter(event)
```

Comment:
internally nothing has really changed but from the public API perspective I don't want to expose users to concepts such as providerlogs or document logs, and thus I'm introducing event presenters to filter this information out